libuuu/config.cpp

```diff
@@ -121,9 +125,9 @@ ConfigItem * Config::find(uint16_t vid, uint16_t pid, uint16_t ver)
 	return nullptr;
 }
 
-Config Config::find(const string &pro)
+Config Config::filter(const string &pro)
 {
-	Config items;
```

Comment:
Thank you for your patch.  Look like Config::find(const string &pro) is never used. we can remove it.

Reply:
Dropped :heavy_check_mark: